packages/cli/src/controllers/users.controller.ts

```diff
@@ -64,23 +69,20 @@ export class UsersController {
 
 	private removeSupplementaryFields(
 		publicUsers: Array<Partial<PublicUser>>,
-		listQueryOptions: ListQuery.Options,
+		listQueryOptions: UsersListFilterDto,
 	) {
-		const { take, select, filter } = listQueryOptions;
+		const { select } = listQueryOptions;
 
 		// remove fields added to satisfy query
+		const fields = select as Array<keyof User>;
```

Comment:
Why the cast here ?

Reply:
Seems like my IDE didn't update during development, this cast is not needed.